src/devices/Common/MathExtensions.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device
+{
+    /// <summary>
+    /// Implementations of some functions missing in older .NET versions
+    /// </summary>
+    public static class MathExtensions
+    {
+        /// <summary>
+        /// Returns val, limited to the range min-max (inclusive)
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static double Clamp(double val, double min, double max)
+        {
+#if NETSTANDARD2_0
```

Comment:
While I'm fine with this for now, an alternative would be to have two implementations of this file that are conditionally compiled, one for the downlevel implementation, and the other one that just forwards the calls to Math to avoid all these ifdefs.